### DISCUSSION TITLE
Is it possible to perform transforms.Spacing on gpu?

### DISCUSSION THREAD

**Discussion by grudloff** [Project-MONAI/tutorials#1521] [Q&A] [answered]
And if it is possible would it lead to a speedup?

**Comment by ericspod** [comment, marked as answer]
`Spacing` will work using CUDA if the input tensor is on a GPU device already. It should give a speed up if the rest of your transform sequence is optimal, the usual places where time is list is converting tensors or moving them to and from devices so avoiding these is important.
